pandas/tests/frame/test_apply.py

```diff
@@ -1012,3 +1012,18 @@ def test_non_callable_aggregates(self):
         expected = df.size
 
         assert result == expected
+
+    def test_frame_row_and_column_aggregates(self):
+        df = DataFrame({'B': [4, 5, 6, 7, 8],
+                        'C': [7, 8, 9, 10, 11]},
+                       index=[1, 2, 3, 4, 5])
+
+        assert_series_equal(df.agg('min', axis=1),
+                            Series({1: 4,
+                                    2: 5,
+                                    3: 6,
```

Comment:
also add a test with mixed dtypes (note that they will generally be excluded on the agg so this works ok)